src/variable.jl

```diff
@@ -220,17 +222,36 @@ mutable struct Tau <: Variable
     data::Vector{Float64}
     λ::Float64
     β::Float64
-    function Tau(β=1.0, λ=0.5, size=MaxOrder)
-        t = [β / 2.0 for i = 1:size]
+    function Tau(β = 1.0, λ = 0.5, size = MaxOrder)
+        t = [β * (i - 0.5) / size for i = 1:size] #avoid dulication
         return new(t, λ, β)
     end
 end
 
+mutable struct Continuous <: Variable
+    data::Vector{Float64}
+    λ::Float64
+    lower::Float64
+    range::Float64
+    function Continuous(bound, λ = nothing, size = MaxOrder)
+        lower, upper = bound
+        @assert upper > lower
+        @assert isnothing(λ) || (0 < λ < (upper - lower))
+        t = [lower + (upper - lower) * (i - 0.5) / size for i = 1:size] #avoid dulication
```

Comment:
why is t initialized here this way while in sampler.jl the create!() initialized the variable randomly?

Reply:
There is no access to the user-defined  rng in this function.